### DISCUSSION TITLE
blog/introducing-meshtastic-solutions/

### DISCUSSION THREAD

**Discussion by giscus** [meshtastic/Discussions#18] [Blog Post Comments]
# blog/introducing-meshtastic-solutions/

Meshtastic Solutions is a new initiative designed to support the growth and long-term success of the open source Meshtastic project

https://meshtastic.org/blog/introducing-meshtastic-solutions/

**Comment by Allanusa** [comment]
Hello my name is Allan.

I have purchased 16 of the LoRa. I have read the setup instructions and watched a number of videos. I am have problems with the android app being able to text. The app will connect I can make changes, but I am not able to text. The same thing going on with the Mesh web site as well. I run the device through the serail flash and I see two error lines. This goes for most devices. I am not sure what the error are on the Node t114. It too how ever will not text. Like I said I have read your instructions, watched numerous videos, tried six different LoRa devices, three USB cables, used three different android devices, updated, unload, and reload all drives, software, and apps.  Is there something that will help me get this project completed. Please point me in the right direction.

Thank you for you time.
Allan

**Reply by rcarteraz** [reply]
You'll probably be better served creating your own post here: https://github.com/orgs/meshtastic/discussions
